packages/argo-run/src/dev.ts

```diff
@@ -98,3 +124,18 @@ function getOpenUrl(args: string[]) {
   const openArg = namedArgument('open', args);
   return (openArg ?? '').startsWith('http') ? new URL(openArg!) : undefined;
 }
+
+function readConfig() {
```

Comment:
suggestion 1: support `extension.config.json` also

suggestion 2: support an `extension_config` object inside `package.json`

Reply:
yeah, I thought about this and was very close to just using https://github.com/davidtheclark/cosmiconfig, however this file also needs to be read from the ruby cli (for the `push` command), so in the interest of simplicity I kept it for `yml` for now.